<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editKeyset</class>
 <widget class="QDialog" name="editKeyset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View and edit a keyset</string>
  </property>
  <widget class="QToolButton" name="toolButtonSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save changes
and close</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonDontSave">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close without
saving changes</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>661</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>User private key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditUserPrivKey">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User SSH public key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditUserSSHKey">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Judge private key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditJudgePrivKey"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Judge SSH public key</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditJudgeSSHKey">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Keyset label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditLabel"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
